habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -106,6 +107,16 @@ def get_local_surface_center(
         Returns the center of receptacle surface in local space
         """
 
+    def check_if_point_on_surface(
+        self,
+        sim: habitat_sim.Simulator,
+        point: mn.Vector3,
+        threshold: float = 0.05,
+    ) -> bool:
+        """
+        Check if point lies within a `threshold` distance of the receptacle's surface
+        """
```

Comment:
don't we need a `pass` command here?

Reply:
Yeah, it is not needed if we have a docstring - [check this](https://stackoverflow.com/questions/46146044/python-s-empty-function-does-not-require-a-pass-statement). I saw it being done this way in other places in this file.

Reply:
not with the comment, but good for code style i think